.storybook/preview.tsx

```diff
@@ -5,13 +5,35 @@ import type { Preview } from '@storybook/react'
 
 import { fakePublishableKey, fakeUserIdentifierKey } from './seed-fake.js'
 
-const publishableKey =
-  process.env['STORYBOOK_SEAM_PUBLISHABLE_KEY'] ?? fakePublishableKey
-
-const userIdentifierKey =
-  process.env['STORYBOOK_SEAM_USER_IDENTIFIER_KEY'] ?? fakeUserIdentifierKey
+const isProd = process.env['NODE_ENV'] === 'production'
+const useFake = process.env[']STORYBOOK_SEAM_ENDPOINT'] == null
```

Comment:
Is `isProd` & `useFake` mutually exclusive?

Reply:
No, and I expect these conditionals will be short lived anyway. Is prod is just if it's going to be deployed. We don't have a great way to detect the fake, but the way thing are setup, the fake is always run by storybook and the endpoint port is random, so if you override the endpoint you would not be using the fake, at least not the one provided here.

Reply:
Hmm, this condition is not quite right though, we will need to add the env var back for now.